AlfrescoApp/View Controllers/FileFolder View Controller/BaseCollectionViewFlowLayout.m

```diff
@@ -19,15 +19,18 @@
 #import "BaseCollectionViewFlowLayout.h"
 #import "BaseLayoutAttributes.h"
 
+static CGFloat const itemSpacing = 10.0f;
+
 @interface BaseCollectionViewFlowLayout ()
 
-@property (nonatomic) CGFloat width;
+@property (nonatomic) CGFloat collectionViewWidth;
+@property (nonatomic) CGFloat thumbnailWidth;
 
 @end
 
 @implementation BaseCollectionViewFlowLayout
 
-- (instancetype)init
+- (instancetype)initWithNumberOfColumns:(NSInteger)numberOfColumns itemHeight:(CGFloat)itemHeight shouldSwipeToDelete:(BOOL)shouldSwipeToDelete
```

Comment:
Nothing wrong with this particularly, but I'd set defaults and keep it to an `init` method. Unless these parameters are changed a lot within the app it's going to need a lot of duplicate calling code to set this controller up. Public properties can be set after `init` to change the defaults.